### DISCUSSION TITLE
SK 0.3.8, Does this version's nuget package automatically choose the ARM64.

### DISCUSSION THREAD

**Discussion by laultman** [StereoKit/StereoKit#787] [Q&A] [answered]
@maluoi I am compiling an app for HoloLens 2 on an Intel computer, targeting Release for ARM64. The question is does this alone cause the code to compile for ARM64 and use the correct packages of SK?

My use case is a custom Teams service integrated into my HoloLens 2 app. The Teams integration runs like a charm on the Intel machine but is dog-slow on the HoloLens 2. It is the same code, one compiled for debug AnyCPU running on the Intel i7 Surface Pro 7.

What I am seeing is solid, no glitching holograms. So, it seems that the framerate is fine. However, inputs seem to be updating at about 1 second intervals. So, for example, my hand pose is captured and is visible and remains fixed in that spot for approximately 1 second then a new input position is captured.

The only common item between the app/SK and Teams API is the camera. However, on the Surface running the app is flawless. Obviously, Microsoft knows how to do this. The Remote Assist app uses Teams as the transport. Any clues would be appreciated.

**Comment by maluoi** [comment, marked as answer]
If you are targeting ARM64, StereoKit should be using the ARM64 binary. You can verify this by looking at StereoKit's first log line, it includes the binary architecture there.

**Comment by laultman** [comment]
Thanks!
